azure-quantum/azure/quantum/job.py

```diff
@@ -50,40 +52,50 @@ def has_completed(self):
         )
 
     def wait_until_completed(self, max_poll_wait_secs=30):
-        """Keeps refreshing the Job's details until it reaches a finished status.
+        """Keeps refreshing the Job's details until it reaches a finished
+        status.
         """
         self.refresh()
         poll_wait = 0.2
         while not self.has_completed():
-            logger.debug(f"Waiting for job {self.id}, it is in status '{self.details.status}'")
+            logger.debug(
+                f"Waiting for job {self.id}, "
+                + f"it is in status '{self.details.status}'")
             print('.', end='', flush=True)
             time.sleep(poll_wait)
             self.refresh()
-            poll_wait = max_poll_wait_secs if poll_wait >= max_poll_wait_secs else poll_wait * 1.5
+            poll_wait = (max_poll_wait_secs
+                         if poll_wait >= max_poll_wait_secs
+                         else poll_wait * 1.5)
 
     def get_results(self):
-        if not self.results is None:
+        if self.results is not None:
             return self.results
 
         if not self.has_completed():
             self.wait_until_completed()
 
         if not self.details.status == "Succeeded":
-            raise RuntimeError(f"Cannot retrieve results as job execution failed (status: {self.details.status}. error: {self.details.error_data})")
+            raise RuntimeError(f"Cannot retrieve results as job execution "
+                               + f"failed (status: {self.details.status}. "
+                               + f"error: {self.details.error_data})")
 
-        url = urlparse(self.details.output_data_uri) 
+        url = urlparse(self.details.output_data_uri)
         if url.query.find('se=') == -1:
-            # output_data_uri does not contains SAS token, get sas url from service
-            blob_client = BlobClient.from_blob_url(self.details.output_data_uri)
-            blob_uri = self.workspace._get_linked_storage_sas_uri(blob_client.container_name, blob_client.blob_name)
+            # output_data_uri does not contains SAS token,
+            # get sas url from service
+            blob_client = BlobClient.from_blob_url(
+                self.details.output_data_uri)
+            blob_uri = self.workspace._get_linked_storage_sas_uri(
+                blob_client.container_name, blob_client.blob_name)
             payload = download_blob(blob_uri)
```

Comment:
I think we need import download_blob from azure.storage.blob for this code path to work.

Reply:
I think we should run pycodestyle on this repo and it will point out pep8 style issues as well.

Reply:
To autofix issues 'black' is a tool to help. It isn;t 100% pep8 compliant but there is way around it